plugin/pkg/scheduler/algorithm/priorities/interpod_affinity.go

```diff
@@ -222,10 +222,11 @@ func (ipa *InterPodAffinity) CalculateInterPodAffinityPriority(pod *v1.Pod, node
 
 	// calculate final priority score for each node
 	result := make(schedulerapi.HostPriorityList, 0, len(nodes))
+	maxPriorityFloat64 := float64(schedulerapi.MaxPriority)
 	for _, node := range nodes {
 		fScore := float64(0)
 		if (maxCount - minCount) > 0 {
-			fScore = float64(schedulerapi.MaxPriority) * ((pm.counts[node.Name] - minCount) / (maxCount - minCount))
+			fScore = maxPriorityFloat64 * ((pm.counts[node.Name] - minCount) / (maxCount - minCount))
```

Comment:
It seems only used for once ...

Reply:
+1 to @resouer's point. These are used only once.

Reply:
a little confused :confused:

Reply:
i mean this type casting happens frequently in a loop, is there anything wrong with my understanding? @resouer @bsalamat :sweat:

Reply:
Oh I see, the title of PR is misleading. Could you please benckmark if this fix is useful? For example, with `go test -bench=.`

Reply:
I would be very surprised if the compiler did not hoist casting of a constant out of the loop. I don't think this has a real impact on performance.